source/applications/dependency-injection.md

```diff
@@ -191,11 +191,17 @@ export default Ember.Component.extend({
 
 ## Factory Instance Lookups
 
-The vast majority of Ember registrations and lookups are performed implicitly.
+You rarely need to write code to look up objects in an Ember application because
```

Comment:
Something like

"In some cases, fetching an instantiated factory from the running application can be useful. The `lookup` method on `Ember.ApplicationInstance` objects provides this functionality. It takes an argument of a factory name, and returns an appropriate object. The application instance object is passed to Ember's instance initializer hooks, and can additionally be referenced as the "owner" of any object that was itself instantiated by the application instance object."